core/templates/services/math-interactions.service.ts

```diff
@@ -61,9 +62,24 @@ export class MathInteractionsService {
         'not valid.';
     }
     if (errorMessage.indexOf('is not a valid postfix operator.') !== -1) {
+      let humanReadableOperator = (
+        errorMessage[0] === '*' ? '×' :
+        errorMessage[0] === '/' ? '÷' :
+        errorMessage[0]);
+      errorMessage = (
+        'Your answer seems to be missing a number after the ' +
+        humanReadableOperator + ' operator.');
+    }
+    if (errorMessage === 'Not a prefix operator.') {
+      let humanReadableOperator = (
+        expressionString[0] === '*' ? '×' :
+        expressionString[0] === '/' ? '÷' :
+        expressionString[1] === '*' ? '×' :
+        expressionString[1] === '/' ? '÷' :
+        expressionString);
```

Comment:
This last one seems like a weird default for humanReadableOperator ... what if the error appears later on in the string?

Reply:
Yeah, @seanlip true!
 See, Probably [this](https://github.com/jiggzson/nerdamer/blob/28d5ca348ad881b765615e56dacfb82521050ee2/nerdamer.core.js#L6510) function in nerdamer is giving that error Message `Not a prefix operator` So, In that case, how can we update our error message for any string in our file, Can you guide me?

Reply:
Hey @seanlip 
`    if (errorMessage === 'Not a prefix operator.') {
      let i : number
      for (i = 0; i <expressionString.length; i++){
        if(expressionString[i] === '*' || expressionString[i] === '/'){
          let humanReadableOperator = (
            expressionString[i] === '*' ? '×' :
            expressionString[i] === '/' ? '÷' :
            expressionString);
            errorMessage = (
            'Your answer seems to be missing a variable/number before the ' +
            humanReadableOperator + ' operator.');
        }
      }
    }`
    
   This piece of code is working fine and fixing this issue.
   PTAL!